src/main/java/org/jvnet/hudson/test/RealJenkinsRule.java

```diff
@@ -909,6 +909,21 @@ public X509Certificate getRootCA() {
         return rootCA;
     }
 
+    /**
+     * @return the current autogenerated root CA in PEM format or null if {@link #https()} has not been called.
```

Comment:
`@link` to/from `#cert`?

Reply:
thanks for the comment.
something wasn't feeling quite right about the javadoc of this😅, feels more correct now😌